events/code_review_events/bugbug_utils.py

```diff
@@ -29,6 +29,12 @@
 
 class BugbugUtils:
     def __init__(self, phabricator_api):
+        self.phabricator_deployment = (
```

Comment:
I would prefer using `APP_CHANNEL` here or a dedicated secret instead of checking the phabricator url

Reply:
The problem with APP_CHANNEL is that sometimes we point testing to look at Phabricator prod and we wouldn't support that

Reply:
I've switched to a secret.